ui/panels-plugin/src/plugins/line-chart/tooltip/tooltip-model.ts

```diff
@@ -13,13 +13,14 @@
 
 import { FocusedSeriesArray } from '../utils/focused-series';
 
-export const TOOLTIP_MIN_WIDTH = 360;
+export const TOOLTIP_MAX_WIDTH = 700;
+export const TOOLTIP_MAX_HEIGHT = 600;
 
 export const TOOLTIP_MAX_ITEMS = 50;
 
 export const TOOLTIP_DATE_FORMAT = new Intl.DateTimeFormat(undefined, {
   year: 'numeric',
-  month: 'numeric',
+  month: 'short',
```

Comment:
any objections to using the `short` format here? I think it reads nice over all, but would be good to hear the opinions of others.

Reply:
I like it!

Reply:
Looks good to me!

Reply:
It looks even better like that ! So cool thanks for doing that :)